### DISCUSSION TITLE
how to make "py" files open?

### DISCUSSION THREAD

**Discussion by wollframium** [felixse/FluentTerminal#996] [General]
I'm new to terminal programs, I have scripts that I run on double click "filename.py"

after that, the python console opens and the script is executed in it.

but since you need to run a lot of copies of the script, it looks not very convenient....

![image](https://user-images.githubusercontent.com/65495044/152621793-018293c0-ec7a-48fb-b034-25b4b9244d92.png)

therefore, I decided to look for a solution in which it would be in the form of tab. 

I really liked this application, it's better than Microsoft made the microsoftterminal, because I couldn't even install it because the Windows version doesn't fit)
but I am faced with the fact that I can not redirect the launch of "files.py" to flute.exe
 i tried creating "cmd" "bat" "vbs" "ps1" and using
flute.exe run
in the content of these files in different ways indicated something like "py filename.py"

gave these errors
![image](https://user-images.githubusercontent.com/65495044/152622628-e324d55d-b40b-4983-918d-ee0c49510b97.png)

maybe i did something wrong here? I don’t know what the working folder is, so I installed the one where the script is, and I don’t know what the arguments are, so I left it empty
![image](https://user-images.githubusercontent.com/65495044/152622712-3f49a305-be13-440e-9b16-0213ca52c896.png)

**Comment by hanskokx** [comment]
python file.py

**Comment by wollframium** [comment]
> python file.py

this is understandable so it works, I checked, but I would not want to manually open 100 copies. everyday.

**Comment by hanskokx** [comment]
Maybe you need to use a loop to open everything in a directory and run them 
as background tasks?

**Comment by wollframium** [comment]
> Maybe you need to use a loop to open everything in a directory and run them as background tasks?

something like that?

Set ws = CreateObject("Wscript.Shell")
ws.run "cmd /c python *название файла*.py",0

this is not very convenient. although the script works without failures and does not require control, it is sometimes nice to see what this or that bot is doing.


and when there are 100 copies, too many windows come out and they are all called not by the names of the bots but by "c: windows \ py"
it would be more convenient to run 10 fluent windows, and in each 10 tabs) I just don’t understand how to do it and what would be done automatically and not manually

but thanks anyway for answering
